snmp/datadog_checks/snmp/data/default_profiles/tp-link.yaml

```
@@ -0,0 +1,38 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "tp_link"
```

```suggestion
        value: "TP-Link"
```

Since the vendor value is used as field, ideally, we should use a human readable form.

Looking at the TP-Link website, they seems to use `TP-Link`